content/1.3-migration.md

```diff
@@ -0,0 +1,73 @@
+## Migrating to use imports everywhere
+
+Never use a symbol without importing it (i.e. no more psuedo-globals!).
+
+In general from core Meteor libraries your code should remain unchanged, and you just add some lines like:
+
+```js
+import {Meteor} from 'meteor/meteor';
+import {ReactiveVar} from 'meteor/reactive-var';
+```
+
+With the following exceptions
+
+
+1. Use `import {Collection, Cursor, ObjectID} from 'meteor/mongo'`
+
+2. Use `import {MeteorServer} from 'meteor/ddp-server'` and call:
+  1. `MeteorServer.publish()`
+  2. `MeteorServer.methods()`
+
+3. Use `import {MeteorConnection} from 'meteor/ddp-client'` and call:
+  1. `MeteorConnection.apply()`
+  1. `MeteorConnection.call()`
+  1. `MeteorConnection.disconnect()`
+  1. `MeteorConnection.onConnection()`
+  1. `MeteorConnection.reconnect()`
+  1. `MeteorConnection.status()`
+  1. `MeteorConnection.subscribe()`
+
+4. Use `import {connect} from 'meteor/ddp-client'` rather than `DDP.connect`;
+
+5. Use `import {Accounts} from 'meteor/accounts-base'` and (on the client) use:
+  1. `Accounts.user()` instead of `Meteor.currentUser()`
+  2. `Accounts.userId()` instead of `Meteor.userId()`,
+  3. `Accounts.loggingIn()` insead of `Meteor.loggingIn()`,
+  4. `Accounts.logout()` instead of `Meteor.logout()`,
+  5. `Accounts.logoutOtherClients()` instead of `Meteor.logoutOtherClient()`
+  6. `Accounts.users` instead of `Meteor.users`
```

Comment:
This looks like a case of maybe we should export `Users` directly, so it looks more like a normal collection? Not sure, and not sure if this would break some stubbing packages